Makefile

```diff
@@ -221,6 +221,69 @@ uninstall:
 	rm -f $(INSTALL_INCLUDE_DIR)/sqlite-vec.h
 	ldconfig
 
+# ███████████████████████████████ ANDROID SECTION ███████████████████████████████
+
+# Define Android NDK paths (update these to your actual NDK path)
+# NDK_PATH ?= /path/to/android-ndk
```

Comment:
ANDROID_NDK_HOME needs to be defined in case you want to compile libraries for Android.